gabc/details.html

```diff
@@ -265,6 +266,14 @@ <h3 id="comments">Comments</h3>
 
 <p>All the characters on a line after a <code>%</code> will be ignored.</p>
 
+<h3 id="elision">Elision</h3>
+
+<p>Starting from version 4.1, syllable text in <code>&lt;e&gt;</code> markup is considered as an elision, a syllable melt in another. This text will be output in its own style (<code>elisionstyle</code>, italic and small by default), and will be ignored by vowel finding algorithm. For example <code>cla&lt;e&gt;ra&lt;/e&gt;(gh)</code> gives:</p>
```

Comment:
"a syllable melt in another" isn't colloquial English.  At the very least the verb should be a past participle ("melted") and the preposition "into", but even then I'm not sure that it's the right verb.  I would say "elided", but I also realize that said verb doesn't really do much as a definition for "elision".  If I come up with a better explanation I'll let you know.